content/docs/start/data-and-model-versioning.md

```diff
@@ -204,11 +201,25 @@ latest version:
 Let's say we obtained more data from some external source. We can pretend this
 is the case by doubling the dataset:
 
+<toggle>
+<tab title="Mac/Linux">
+
 ```dvc
 $ cp data/data.xml /tmp/data.xml
 $ cat /tmp/data.xml >> data/data.xml
 ```
 
+</tab>
+<tab title="Windows">
+
+```powershell
+copy data\data.xml ...\tmp\data.xml
+type .../tmp/data.xml >> data\data.xml
```

Comment:
Here I did change the commands so I also had to use some non-specific `...\tmp` path to the temp folder for now.

Reply:
does it have env var by chance? %TEMP% or something?

Reply:
Yeah it just implies we need two change many sample blocks into tab toggles just because of the path to tmp dir. Is it worth it?

Reply:
AFAIK windows doesn't have a true equivalent by default. Linux `/tmp/` is a volatile ramdisk, while windows `%TEMP%` and `%TMP` are not :(

Reply:
...which further complicates being perfectly Windows friendly but I guess that detail we could ignore.